### ISSUE TITLE
Logout breaks

### ISSUE THREAD

**Issue by agracey** [SUSE-Developer-Community/explore.suse.dev#2]
Currently log out doesn't redirect with correct state

**Comment by feri** [comment]
When hitting "Logout" Wordpress sets the Location header in the response as follows:

`Location: https://exploresusesso.auth.us-west-2.amazoncognito.com/logout?id_token_hint=eyJraWQiOiJ3bk94NUN2OHhnbklGcVA3Y1pwZjRpWG5mbXI0YkV1WmYrckpEc3RidmRZPSIsImFsZyI6IlJTMjU2In0.eyJhdF9oYXNoIjoiYzhvT0JTbTNrOFc2eDJTZEhZLV9LUSIsInN1YiI6ImY4ZjFjZDhhLWM0NzQtNGY2Zi05YWQ1LTVkMmEzNmZjNjcyNiIsImF1ZCI6IjJzMXFvNWxramdiYWJsZ3AycTU3Z3NjYXNnIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInRva2VuX3VzZSI6ImlkIiwiYXV0aF90aW1lIjoxNTgyMjMyNDU0LCJpc3MiOiJodHRwczpcL1wvY29nbml0by1pZHAudXMtd2VzdC0yLmFtYXpvbmF3cy5jb21cL3VzLXdlc3QtMl9DTTczUFp6V20iLCJjb2duaXRvOnVzZXJuYW1lIjoiZjhmMWNkOGEtYzQ3NC00ZjZmLTlhZDUtNWQyYTM2ZmM2NzI2IiwiZXhwIjoxNTgyMjM2MDU0LCJpYXQiOjE1ODIyMzI0NTQsImVtYWlsIjoiZmVyZW5jLnN6ZWtlbHlAc3VzZS5jb20ifQ.R8Ogk3zhaAeWWgBMjjB4Qt_iclQaBIJ0Lpm-p5bJawxAvdn0faK11AY6mDHa6ELI5T2XiYKbDw56Qlrnv3sBs8eoMPvk9WKvGR2Q4PhpjEldlLI9Lvi8p7uiWBKPA1F5Qus5IoDGp8yVlBLz5D1Ad7OLF8LrnjeF_DlQsFltCVyAECe6oLFHsQgLqHsX4Wnj2U54OBqixsVEIp5vbpYivqrSAbug07ESleRnOqy5TUhu4hNblXS8b-Ca0eZK5deXGy2RIPiyajKPJ9i3glpSK53CAMx8T_K0Xb2ip6HIaTp7tz_DObK59fkH92HHDo54ImH9dLtSKQDHlWBKnpbDpA&post_logout_redirect_uri=https%3A%2F%2Fwww.explore.suse.dev%2Fwp-login.php%3Floggedout%3Dtrue%26wp_lang%3Den_US
`

When the browser receives this response it will immediately load the Cognito URL. I believe the value of this header should be extended with the "client_id" so that Cognito could verify that too. 

I will check if we could convince the OpenID plugin to extend the Location header upon logout.

**Comment by agracey** [comment]
The plugin that I used is here: https://github.com/daggerhart/openid-connect-generic

We could fork it to add any improvements back.

**Comment by agracey** [comment]
As a heads up, I had edited the redirect on successful login to home_url() instead of admin_url(). I should have forked and installed from git instead of the marketplace so I could track edits.

**Comment by feri** [comment]
![screenshot-exploresusesso auth us-west-2 amazoncognito com-2020 03 05-21_21_02](https://user-images.githubusercontent.com/232490/76017442-48ebd600-5f27-11ea-96a1-b3de43939be1.png)

**Comment by feri** [comment]
The new reg and login / logout flow has been rolled out, which eliminates this issue.
